bookwyrm/views/admin/user_admin.py

```diff
@@ -95,3 +95,18 @@ def post(self, request, user):
                 form.save(request)
         data = {"user": user, "group_form": form}
         return TemplateResponse(request, "settings/users/user.html", data)
+
+
+@method_decorator(login_required, name="dispatch")
+@method_decorator(
+    permission_required("bookwyrm.moderate_user", raise_exception=True),
+    name="dispatch",
+)
+class ActivateUserAdmin(View):
+    """activate a user manually"""
+
+    def post(self, request, user):
```

Comment:
You can disable the pylint error by adding `# pylint: disable=unused-argument` above this line.